### ISSUE TITLE
Button Contrast Ratio Issues in Footer

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#991]
The button text/link is received from the parent footer styles resulting in bad color contrast ratios. I have not tested but it may affect alert and alert link styles as well.

admin/config basic site settings footer content

**Comment by joewhitsitt** [comment]
Resources towards uiowa_bootstrap should be avoided at this time.
